setup.py

```diff
@@ -21,7 +21,7 @@ def find_stub_files(name):
 
 dependencies = [
     "mypy>=0.790",
```

Comment:
Should we use the same `mypy` version as in `django-stubs`?

Reply:
Sure